src/pages/Login.jsx

```diff
@@ -1,39 +1,83 @@
-import React from 'react'
-import { Link } from 'react-router-dom'
+import React, { useState } from 'react'
+import { Link, useNavigate } from 'react-router-dom'
 import '../styles/login-and-signup.css'
 import Header from './Header'
+import { useAuth } from '../contexts/auth-context'
+import { loginHandler } from '../utilities/auth'
+
+const formInitialState = {
+    email: '',
+    password: '',
+    rememberMe: false,
+}
 
 const Login = () => {
+    const [formData, setFormData] = useState(formInitialState);
+    const { email, password, rememberMe } = formData;
+    const { saveUserInfo } = useAuth();
+    const navigate = useNavigate();
+
+    const inputHandler = (e) => {
+        setFormData((prevFormData) => ({
+            ...prevFormData,
+            [e.target.name]: e.target.value,
+        }))
+    }
+    const handleToggle = (e) => {
+        setFormData((prevFormData) => ({
+            ...prevFormData,
+            [e.target.name]: e.target.checked
+        }))
+    }
   return (
     <>
         <Header />
-        <main class="login-page">
-            <div class="login-card flex-column">
-                <p class="text-l login-heading">Login</p>
-                <div class="login-input-field flex-column">
-                    <label for="email" class="email-label">Email address</label>
+        <main className="login-page">
+            <form className="login-card flex-column">
+                <p className="text-l login-heading">Login</p>
```

Comment:
one thing i would add to this form element is an `onSubmit` function, what you can do is pass the `loginHandler` in the form element as `onSubmit` and remove the `onClick` form login button and add `type= "submit"` to the button. If you do this you can also submit the form by clicking `enter` key. one of my favourite features to add in forms.
```js
<form className="...." onSubmit={loginHandler(................)}>
.....
  <button type="submit"> Login </button>
</form>
```

Reply:
Sure, will make the changes